### DISCUSSION TITLE
[Enhancement] [select] 可不可以把value-on-clear默认值改为'' 空字符串

### DISCUSSION THREAD

**Discussion by OrensDon** [element-plus/element-plus#16876] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

这是一个老的js引入的项目，无法修改全局配置，且valueOnClear的配置是无效的。
之前一直默认是空字符串，更新到2.7之后，全崩啦！！！像el-select点击清空后，值全变成了undefined传到后端啦！！！
加新配置项是好的，但是能不能不要改原来的默认值，拜托！！！

### Additional comments

拜托！！！

**Comment by OrensDon** [comment]
不好意思，默认值不是2.7的更新，配置项是。valueOnClear是有效的，但是也挺麻烦，因为每个页面都要加一行

**Reply by kooriookami** [reply]
https://element-plus.org/zh-CN/component/config-provider.html#%E7%A9%BA%E5%80%BC%E9%85%8D%E7%BD%AE
可以包一层el-config-provider

**Comment by warmthsea** [comment]
纯文字的描述很难给你帮助及参考

**Comment by OrensDon** [comment]
非常感谢，已经解决了Sent from my iPhoneOn May 17, 2024, at 15:56, kooriookami ***@***.***> wrote:﻿
https://element-plus.org/zh-CN/component/config-provider.html#%E7%A9%BA%E5%80%BC%E9%85%8D%E7%BD%AE
可以包一层el-config-provider
